external/build/lib/dts-parse.js

```diff
@@ -807,6 +807,7 @@ module.exports = async function parse({silent, sources, mode}) {
   /** @type {typedoc.TypeScript.CompilerOptions} */
   const typescriptOptions = {
     declaration: true,
+    esModuleInterop: true,
```

Comment:
The `webpack` dependency that `workbox-webpack-plugin` imports is still CommonJS, so this flag is needed.